docs/modules/ref_thorntail-maven-plugin-configuration-options.adoc

```diff
@@ -262,18 +262,19 @@ A file path where to store the `stdout` output instead of sending it to the `std
 |===
 
 useUberJar::
-If true, the `-thorntail.jar` file specified at `${project.build.directory}` is used.
+If specified, the `-thorntail.jar` file located in `${project.build.directory}` is used.
```

Comment:
Please change to _...located in the..._

Reply:
In CCS, there is an informal convention that if a variable name already contains the word _directory,_ it's not necessary to repeat it, but I guess it can stay as it is as well.